letsdrill.rb

```diff
@@ -1,24 +1,42 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  if integer >= 90
```

Comment:
if `>=` isn't sufficient and you need a 'lower bound', you can also use ranges

example:
```ruby
case integer
when 80..90
  ...
end
```